src/PhpBrew/Build.php

```diff
@@ -98,6 +98,19 @@ public function setOSRelease($osRelease)
         $this->osRelease = $osRelease;
     }
 
+    public function setPkgConfigPath($path)
+    {
+        $value = getenv('PKG_CONFIG_PATH');
+        if ($value === "") {
+            putenv('PKG_CONFIG_PATH='.$path);
+        } else {
+            $pkgConfigPaths = explode(PATH_SEPARATOR, $value);
```

Comment:
If the original `PKG_CONFIG_PATH` is empty, it will result in the array containing one empty string instead of an empty array. So the resulting `PKG_CONFIG_PATH` will be `:/new/path` instead of `/new/path`.

Reply:
I will remove the empty string element first.

Reply:
If you rework it as suggested in https://github.com/phpbrew/phpbrew/pull/1085#discussion_r358589687, it won't be needed.

Reply:
Done.